redisinsight/ui/src/components/virtual-table/VirtualTable.tsx

```diff
@@ -436,7 +436,7 @@ const VirtualTable = (props: IProps) => {
               data-testid="score-button"
               style={{ justifyContent: column.alignment }}
             >
-              <Text size="m" className={cellClass}  variant="semiBold">
+              <Text size="m" className={cellClass}>
```

Comment:
why? based on designs it should be semibold

Reply:
Interesting, I don't remember making such changes. Maybe something messed up since we both touched this. I'll rebase to sync the latest changes and we'll see, maybe it will be automatically resolved.